src/assets/js/src/mailoptin.js

```diff
@@ -705,7 +705,7 @@ var mailoptin_optin = {
 
         if(optin_type === 'bar' || optin_type === 'lightbox' || optin_type === 'slidein') {
             var optin_sound_config = optin_config.optin_sound;
```

Comment:
@ShoboySnr do the type of check here before setting the variable to prevent any undefined error from being triggered.

something like

`var optin_sound_config = typeof optin_config.optin_sound !== 'undefined' ? optin_config.optin_sound : '';`

Reply:
okay